tests/components/accessibility/autofocus-test.js

```diff
@@ -0,0 +1,44 @@
+jest.dontMock('../../../js/components/accessibility/autofocus');
+
+const Autofocus = require('../../../js/components/accessibility/autofocus');
+
+describe('Autofocus', () => {
+
+  let autofocus, controllerMock, popoverObj;
+  beforeEach(() => {
+    popoverObj = { 'test': true, wasTriggeredWithKeyboard: jest.fn() };
+    controllerMock = {
+      toggleButtons: {
+        popover: popoverObj,
+        test: false,
+      },
+      state: {
+        showPopoverOptions: {
+          showPopover: true,
+          autoFocus: true,
+        }
+      }
+    };
+    autofocus = new Autofocus(controllerMock);
+  });
+
+  it('tests function getToggleButtons', () => {
+    const popover = autofocus.getToggleButtons('popover');
+    const notPopover = autofocus.getToggleButtons('notPopover');
+    expect(popover).toEqual(popoverObj);
+    expect(notPopover).toEqual({});
+  });
+
+  it('tests function setToggleButtons', () => {
```

Comment:
please make test names more verbose, like `should set toggleButtons`

Reply:
\+